README.md

```diff
@@ -41,6 +41,17 @@ To call a particular endpoint, use
 
     RESIM_CLIENT_ID=<client ID> RESIM_CLIENT_SECRET=<client secret> resim create project <flags> 
 
+### Autocomplete
+
+If you would like resim commands to autocomplete you can generate autocomplete scripts using e.g.
+
+    resim complete bash > resim_bash_completion
```

Comment:
```suggestion
    resim completion bash > resim_bash_completion
```

Reply:
If you do this you can omit `completion.go`